log/development.log

```diff
@@ -619,3 +619,15 @@ Migrating to AddTimestampsToMerchants (20170502005406)
   [1m[36mActiveRecord::InternalMetadata Load (0.2ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", :environment], ["LIMIT", 1]]
```

Comment:
@stovermc Let's add the log directory to the gitignore. We don't want these taking up unnecessary space. I'd recommend adding it to your global gitignore.